recipes/nmos-cpp/all/conanfile.py

```diff
@@ -1,9 +1,12 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools import build, files
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
 import json
 import os
 import re
-from conans import ConanFile, CMake, tools
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.50.0"
```

Comment:
export_conandata_patches requires 1.52.0 https://docs.conan.io/en/latest/changelog.html#aug-2022
```suggestion
required_conan_version = ">=1.52.0"
```

Reply:
Thanks!